doc/release-notes/6980-datacite-reservation-upgrade

```diff
@@ -0,0 +1,28 @@
+# Dataverse installatins using Datacite: upgrade action required.
+
+For Dataverses using Datacite v5.0 changes the process of registering the Global Identifier (DOI) for the dataset. Instead of registering it when the dataset is published for the first time, Dataverse will try to "reserve" the DOI when it's created (by registering it as a "draft", using Datacite terminology). When the user publishes the dataset, the DOI will be publicized as well (by switching the registration status to "findable"). 
```

Comment:
```suggestion
For installations using Dataverse v5.x, the process of registering the Global Identifier (DOI) for the dataset changes. Instead of registering it when the dataset is published for the first time, Dataverse will try to "reserve" the DOI when it's created (by registering it as a "draft", using Datacite terminology). When the user publishes the dataset, the DOI will be publicized as well (by switching the registration status to "findable"). 
```

Reply:
And is this really true for all types of PIDs? I couldn't tell from a quick view at the code, but if not, maybe this should be more Datacite-using specific?

Reply:
Hmm. I can change "dataverses" to "installations" if you prefer; but otherwise your proposed commit is changing what  I am actually trying to say, somewhat. We want it to say "dataverses (or installations) **using Datacite**" - because this is Datacite-specific, and I do want to repeat it in this paragraph. That sentence may need a comma - but I don't think it should go where you are  adding it. I meant to say something along the lines of 
"For (Dataverse) installations (that are) using Datacite, (the upgrade to) v5.0 will change the process of ..." 

May not have been my best sentence ever, I'll try improve it.  But not sure I like your version either, sorry.

Reply:
That's fine. I was just concerned by a non-existing "DataCite 5.0" :smile:

Reply:
A missing period? using Datacite. v5.x changes....

Reply:
@qqmyers it should be properly punctuated in the current version, yes.